roles/keystone-defaults/defaults/main.yml

```diff
@@ -15,6 +15,8 @@ keystone:
       distro: /usr/lib/python2.7/site-packages/keystone
       source: /opt/openstack/current/keystone
       package: /opt/openstack/current/keystone
+  distro:
+    python_post_deps: []
```

Comment:
we should probably provide a commented out example so that people can see the expected dictionary structure inside the array ?